CONTRIBUTING.md

```diff
@@ -3,53 +3,45 @@
 The development flow includes:
 
 - [Contributing](#contributing)
-  - [Create workspace, clone sources](#create-workspace-clone-sources)
+  - [Start Workspace](#start-workspace)
   - [Build](#build)
   - [Run tests](#run-tests)
   - [Run](#run)
-  - [Generate Devfile](#generate-devfile)
-  - [Create Workspace](#create-workspace)
   - [Package binaries](#package-binaries)
   - [Push changes, provide Pull Request](#push-changes-provide-pull-request)
   - [Make Release](#make-release)
 
-## Create workspace, clone sources
+## Start Workspace
 
-To create a workpace you can use [devfile](devfile.yaml):
+To start a workpace from the [devfile](devfile.yaml) create a workspace URL by appending the chectl repo to the URL of your Che host. For example, to create a workspace in the openshift sandbox use the URL:
```

Comment:
```suggestion
To start a workpace from the [devfile](devfile.yaml), create a workspace URL by appending the chectl repository to the URL of your Che host. For example, to create a workspace in the OpenShift sandbox, use the following URL:
```

Reply:
FYI there is a typo on workpace --> workSpace
```suggestion
To start a workspace from the [devfile](devfile.yaml) create a workspace URL by appending the chectl repo to the URL of your Che host. For example, to create a workspace in the openshift sandbox use the URL:
```

Reply:
Probably want to capitalize `Sandbox` as seen on https://developers.redhat.com/developer-sandbox